app/Models/User.php

```diff
@@ -31,4 +44,39 @@ class User extends Authenticatable
         'password',
         'remember_token',
     ];
+    /**
+     * Get topics: One to many
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoCommaAfterLast</br>**Severity**: error</br>**File**: app/Models/User.php</br>Comma required after last value in array declaration